db/migrate/20150703084923_move_existing_load_profiles_to_load_curves.rb

```diff
@@ -0,0 +1,14 @@
+class MoveExistingLoadProfilesToLoadCurves < ActiveRecord::Migration
+  def change
+    LoadProfile.all.each do |load_profile|
```

Comment:
Migration fails on af0ca565 as the `LoadProfile` class is now `ProfileCurve`. I think you'll have to drop down to raw SQL to run this migration now. :persevere: 

```
StandardError: An error has occurred, all later migrations canceled:

NameError: uninitialized constant MoveExistingLoadProfilesToLoadCurves::LoadCurve
moses/db/migrate/20150703084923_move_existing_load_profiles_to_load_curves.rb:5:in `block in change'
moses/db/migrate/20150703084923_move_existing_load_profiles_to_load_curves.rb:3:in `change'
Tasks: TOP => db:migrate
```